specification/communication/Communication.JobRouter/tspconfig.yaml

```diff
@@ -0,0 +1,55 @@
+parameters:
+  "python-sdk-folder":
+    default: "{project-root}/azure-sdk-for-python/"
+  "java-sdk-folder":
+    default: "{project-root}/azure-sdk-for-java/"
+  "js-sdk-folder":
+    default: "{project-root}/azure-sdk-for-js/"
+  "csharp-sdk-folder":
+    default: "{project-root}/azure-sdk-for-net/"
+  "service-directory-name":
+    default: "communication"
+  "service-dir":
+    default: "sdk/communication"
+  "python-output-dir":
+    default: "azure/communication/jobrouter/_generated"
+emit: [
+  "@azure-tools/typespec-autorest"
+]
+
+linter:
+  extends:
+    - "@azure-tools/typespec-azure-core/all"
+
+options:
+  "@azure-tools/typespec-autorest":
+    azure-resource-provider-folder: "data-plane"
+    output-file: "{azure-resource-provider-folder}/JobRouter/{version-status}/{version}/communicationservicejobrouter.json"
+    emitter-output-dir: "{project-root}/.."
+    omit-unreachable-types: true
+    examples-directory: "{project-root}/examples"
+  "@azure-tools/typespec-csharp":
+    clear-output-folder: true
+    model-namespace: false
+    emitter-output-dir: "{csharp-sdk-folder}/sdk/{service-directory-name}/{namespace}/src"
+    namespace: Azure.Communication.JobRouter
+    package-dir: "Azure.Communication.JobRouter"
+    save-inputs: true
+  "@azure-tools/typespec-python":
+    package-pprint-name: "\"Communication JobRouter\""
+    emitter-output-dir: "{python-sdk-folder}/sdk/{service-directory-name}/{package-name}"
+    package-dir: "azure-communication-jobrouter"
+    package-name: "{package-dir}"
+    package-version: 1.0.0
+  "@azure-tools/typespec-java":
+    emitter-output-dir: "{java-sdk-folder}/sdk/{service-directory-name}/azure-communication-jobrouter"
+    namespace: com.azure.communication.jobrouter
+    partial-update: true
+  "@azure-tools/typespec-ts":
+    emitter-output-dir: "{js-sdk-folder}/sdk/{service-directory-name}/communication-job-router-rest"
+    generateMetadata: true
+    generateTest: true
```

Comment:
I would recommand to remove the below config because CodeGen would directly detect if we need to generate test and relevant metadata files.

```
generateMetadata: true
generateTest: true
```

Reply:
@MaryGao I've removed them